examples/motor/m3508_steering.cc

```diff
@@ -51,63 +54,89 @@ bool steering_align_detect() {
 
 void RM_RTOS_Init() {
   print_use_uart(&huart8);
-  bsp::SetHighresClockTimer(&htim2);
```

Comment:
Does this only work for typeA board?

Reply:
Yes. The example is written against typeA and not tested on typeC.